examples/criteo_benchmark.py

```diff
@@ -91,18 +91,13 @@ def parse_args():
 proc.add_feature([ZeroFill(replace=True), LogOp(replace=True)])
 proc.add_preprocess(Normalize(replace=True))
 if int(args.freq_thresh) == 0:
-    proc.add_preprocess(
-        Categorify(replace=True, on_host=True, split_out=4, cat_cache="host", out_path=args.out_dir)
-    )
+    proc.add_preprocess(Categorify(replace=True, out_path=args.out_dir))
```

Comment:
Agreed - In fact, I think we should think of datascientist-friendly ways to implicitly set the `on_host` and (renamed) `tree_size` parameters. For exeample, we can just expect the user to pass in estimated cardinality (otherwise we can automatically estimate it ourselves from 1+ partition samples).  Note that @EvenOldridge and I discussed this a bit last week